ui/StatusQ/src/StatusQ/Core/Utils/AmountsArithmetic.qml

```diff
@@ -140,6 +140,16 @@ QtObject {
         return amount1.plus(amount2)
     }
 
+    /*!
+      \qmlmethod AmountsArithmetic::sub(amount1, amount2)
+      \brief Returns a Big number whose value is the subtraction of amount2 from amount1.
+     */
+    function sub(amount1, amount2) {
+        console.assert(amount1 instanceof Big.Big)
+        console.assert(amount2 instanceof Big.Big || Number.isInteger(amount2))
```

Comment:
https://mikemcl.github.io/big.js/#minus allows string as well, is it expected for us to also allow strings here?

Reply:
I don't think we should be too flexible. Better explicitly call from string instead.